mqtt-bridge/src/main/java/io/strimiz/strimzi_mqtt_bridge/core/MqttServerHandler.java

```diff
@@ -0,0 +1,57 @@
+package io.strimiz.strimzi_mqtt_bridge.core;
+
+
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.SimpleChannelInboundHandler;
+import io.netty.handler.codec.mqtt.*;
+
+public class MqttServerHandler extends SimpleChannelInboundHandler<Object> {
+    @Override
+    public void channelActive(ChannelHandlerContext ctx) {
+        System.out.printf("Client " + ctx.channel().remoteAddress() +  " is trying to connect\n");
```

Comment:
need for a logging library and not using `System.out`. We should go with using log4j2 in all classes.

Reply:
Noted sir, in this case I just used sout for rapid testing.
I am working on it.